modules/WebcastPolls/resources/webcastPolls.css

```diff
@@ -0,0 +1,438 @@
+
```

Comment:
we isolated our styles from the others by nesting them all under '.webcastPolls' class